x-pack/plugins/security_solution/server/lib/prebuilt_dev_tool_content/console_templates/enable_host_risk_score.console

```diff
@@ -198,31 +218,51 @@ POST _transform/ml_hostriskscore_pivot_transform_{{space_name}}/_start
 # 9. Create mappings for the destination index of the Host Risk Score latest transform
 PUT ml_host_risk_score_latest_{{space_name}}
 {
-  "mappings":{
-    "properties":{
-      "host.name":{
-        "type":"keyword"
-      },
-      "@timestamp": {
-        "type": "date"
+  "mappings": {
+    "properties": {
+      "host": {
+        "properties": {
+          "name": {
+            "type": "keyword"
+          },
+          "risk": {
+            "properties": {
+              "calculated_score_norm": {
+                "type": "float"
+              },
+              "calculated_level": {
```

Comment:
This field used to be text:
```
     "risk": {
        "type": "text",
        "fields": {
          "keyword": {
            "type":  "keyword"
          }
        }
      },
```
If we change the type, we need to update the UI code.

Reply:
@machadoum Open to suggestions on what the field mappings should be. I just think `risk.keyword` looks a bit awkward. `rule_name` seems to be the only field mapped as `text` for now. I was thinking of changing that to `keyword` too.